types/vscode/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"adjacent-overload-signatures": false,
+		"array-type": false,
+		"arrow-return-shorthand": false,
+		"ban-types": false,
+		"callable-types": false,
+		"comment-format": false,
+		"dt-header": false,
```

Comment:
You cannot disable this rule (as you suspected) - publishing won't work if the header metadata isn't readable.

Reply:
Thanks, resolved.

Is that the only field I should care about for publishing, or are there other fields affecting publishing too?

Reply:
Well, you need the lint rule to actually pass~

Reply:
Oops, sorry. Stupid me.